common-test-lib/src/main/kotlin/com/google/container/tools/test/ContainerToolsRule.kt

```diff
@@ -32,7 +38,7 @@ import org.junit.runners.model.Statement
  *  * Creates an [IdeaProjectTestFixture] and makes it available for tests via property. By default
  *    this text fixture is "light", i.e. instance of [LightIdeaTestFixture].
  */
-class ContainerToolsRule : TestRule {
+class ContainerToolsRule(private val testInstance: Any) : TestRule {
```

Comment:
Can `testInstance` scoped to something else than `Any`?

Reply:
Couldn't see anything more specific, it could be any type of object, we are only interested in its properties and annotations here. What would be a Kotlin way here?

Reply:
If it could be anything then `Any` is appropriate.